src/LodeRunner/src/ValidationTest/Main.cs

```diff
@@ -258,8 +258,16 @@ public async Task<int> RunOnce(Config config, CancellationToken token)
             }
             finally
             {
-                // fire event
-                TestRunComplete(null, new LoadResultEventArgs(startTime, DateTime.UtcNow, config.TestRunId, requestCount, validationFailureCount + errorCount));
+                if (token.IsCancellationRequested)
+                {
+                    // fire event
+                    TestRunComplete(null, new LoadResultEventArgs(startTime, DateTime.UtcNow, config.TestRunId, 0, 0, SystemConstants.TestRunExecutionStoppedMessage));
```

Comment:
might be worth capturing requestcount and failure count inside cancellation event and leave it up to user/ui to determine how to use that data

Reply:
Fixed